kruize-clowdapp.yaml

```
@@ -28,6 +28,26 @@ objects:
           limits:
             cpu: ${KRUIZE_CPU_LIMIT}
             memory: ${KRUIZE_MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /health
+            port: 9000
```

```suggestion
            port: ${AUTOTUNE_SERVER_PORT}
```

IIUC this /health API exposed by Kruize API server, correct? Not via Prometheus server?
So, here I think it requires `AUTOTUNE_SERVER_PORT` value.
For Prometheus, port is 9000.

---

Thanks, change pushed